package.json

```diff
@@ -46,36 +45,36 @@
     "webpack-watch": "webpack --mode development --watch"
   },
   "dependencies": {
-    "@babel/traverse": "^7.7.4",
-    "jest-editor-support": "^27.1.0",
+    "@babel/traverse": "^7.9.5",
+    "jest-editor-support": "^27.1.1",
     "lodash": "^4.17.15",
     "micromatch": "^4.0.2",
-    "tslib": "^1.10.0",
+    "tslib": "^1.11.1",
     "vscode-test-adapter-api": "^1.7.0",
     "vscode-test-adapter-util": "^0.7.0"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.25",
+    "@types/jest": "^25.2.1",
     "@types/lodash": "^4.14.149",
-    "@types/micromatch": "^3.1.1",
+    "@types/micromatch": "^4.0.1",
     "@types/node": "^13.1.4",
-    "@types/webpack": "^4.41.1",
-    "jest": "^24.9.0",
+    "@types/webpack": "^4.41.11",
+    "@types/vscode": "^1.44.0",
+    "jest": "^25.3.0",
     "jest-junit": "^10.0.0",
-    "prettier": "^1.19.1",
+    "prettier": "^2.0.4",
     "ts-jest": "^24.2.0",
-    "ts-loader": "^6.2.1",
-    "ts-node": "^8.5.4",
+    "ts-loader": "^7.0.0",
+    "ts-node": "^8.8.2",
     "tslint": "^5.20.1",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^3.7.3",
     "vsce": "^1.70.0",
-    "vscode": "^1.1.36",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10"
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11"
   },
   "engines": {
-    "vscode": "^1.23.0"
+    "vscode": "^1.44.0"
```

Comment:
I just looked up the version list for Code and I see that the original 1.23 is ancient.  However, if we bump the version too high, will we not prevent some users from being able to use the extension?  Maybe we drop it down a couple of versions just in case?

Reply:
It's what I tested it with. Most people will be on a rolling release, but yes I think putting it back will help someone. Will do later today